charactersheet/charactersheet/models/common/treasure.js

```diff
@@ -24,35 +27,37 @@ function Treasure() {
         return Math.round(total);
     });
 
+    self.totalWeight = ko.pureComputed(function() {
+        var weight = 0;
+        weight += parseInt(self.platinum());
```

Comment:
You need to check for ```null``` or ```empty``` in all these. If one of these is a bad value, it will cause a ```NaN``` to appear.